tcms/testplans/forms.py

```diff
@@ -130,8 +130,9 @@ def clean(self, data, initial=None):
 
 class PlanModelForm(forms.ModelForm):
     class Meta:
-        model = TestPlan
-        exclude = ('author', )
+        def __init__(self):
```

Comment:
It throws a `modelform-uses-exclude` and [here](https://github.com/PyCQA/pylint-django/issues/168) it says that it might become a security issue if it isn't fixed.
But also i don't think i fully understand what `Use explicit fields instead of exclude in ModelForm` means.

Reply:
Read this https://docs.djangoproject.com/en/2.1/topics/forms/modelforms/ and see the examples. Then ping me if they are not clear.

Reply:
Fix this commit so I can cherry-pick